packages/connectivity/src/scp-cf/circuit-breaker.spec.ts

```diff
@@ -31,24 +39,29 @@ describe('circuit breaker', () => {
       .get(/.*/)
       .times(1)
       .reply(200, JSON.stringify({ URL: 'test' }));
-    nock(destinationServiceUri).get(/.*/).times(attempts).reply(400);
+    nock(destinationServiceUri).get(/.*/).times(99).reply(400);
```

Comment:
nitpick: 99 as in 'infinitely often', right? maybe use a constant for clarity.

nitpick 2 (not new in this PR): Are there no default constants for http status codes in nodejs?

Reply:
I am not aware of the default constants.